lua/autorun/client/TrueCulling.lua

```diff
@@ -0,0 +1,68 @@
+jit.on()
+
+CULLFRONT = math.sin(0.57357644) -- Degrees forward angle in sine
+
+function main()
+	allEnts = ents.GetAll()
+	if engine.TickCount()%4 == 0 then
```

Comment:
And yet, you are only using the result of that call, every forth frame?

Reply:
If you want to run this every Nth frame for speed because it doesn't need to be updated every frame, this will technically result in microstutter where every Nth frame is slower. What you can do instead is amortize the checks, something like this:

```for ent,_ in pairs(ents) do
    if ent:EntIndex() % N == FrameNumber() % N then
        (do checks)
    end
end```